octobot_commons/async_job.pxd

```diff
@@ -20,10 +20,10 @@ cdef class AsyncJob:
     cdef object callback                    # Callable
     cdef object job_task                    # Asyncio.Task
     cdef object job_periodic_task           # Asyncio.Task
-    cdef public object idle_task_event      # Asyncio.Event
+    cdef public object idle_task_event
+    cdef public bint should_stop   # Asyncio.Event
 
     cdef bint is_started
-    cdef bint should_stop
     cdef bint is_periodic
```

Comment:
?

Reply:
used in clock synch to know if the job is running

Reply:
Oh ok it was not in the commit diff.

Reply:
(yes, it was another issue XD)

Reply:
I think we should create a method like `is_stopping` instead of reading `should_stop` value